File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        res = [i for i in range(len(nums))]
        for i in range(len(nums)):
            n = 1
            for j in range(len(nums)):
                if i == j:
                    continue
                else:
                    n = n * nums[j]

            res[i] = n

        return res
        

File: Data Structures & Algorithms/duplicate-integer/submission-0.cs
public class Solution {
    public bool hasDuplicate(int[] nums) {
        int count = 0;
        foreach (int item in nums)
        {
            count = 0;
            foreach (int num in nums)
            {
                if (item == num)
                count++;
                if (count > 1)
                {
                    return true;
                }
            }
        }
        return false;
    }
}